go-controller/pkg/types/const.go

```diff
@@ -64,7 +64,8 @@ const (
 	InterNodePolicyPriority               = "1003"
 	HybridOverlaySubnetPriority           = 1002
 	HybridOverlayReroutePriority          = 501
-	DefaultNoRereoutePriority             = 101
+	DefaultNoRereoutePriority             = 102
```

Comment:
did you check to make sure on upgrade we handle updating the current acls from 101->102 that would be in use for egress IP?

Reply:
not yet, will update here once I do

Reply:
verified now :)